src/localisation/lang/en/locale.json

```diff
@@ -41,6 +41,7 @@
     "more-info": "More Info",
     "track": "Track",
     "playlists": "Playlists",
+    "timer": "Set Sleep Timer",
```

Comment:
Please make the key more descriptive, timers could be used in other contexts. E.g. `sleep-timer`.

Reply:
I removed the key 'timer' when resolving the previous issue.